files/default/sql/ddl/1.1.0__initial_tables.sql

```diff
@@ -1112,6 +1112,51 @@ CREATE TABLE `project_team` (
 ) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- table structure for table `schemas`
+--
+
+/*!40101 set @saved_cs_client     = @@character_set_client */;
+/*!40101 set character_set_client = utf8 */;
+CREATE TABLE `schemas` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `schema` varchar(10000) COLLATE latin1_general_cs NOT NULL,
+  `project_id` int(11) NOT NULL,
+  PRIMARY KEY (`id`, `project_id`),
+  CONSTRAINT `project_idx` FOREIGN KEY (`project_id`) REFERENCES `project` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION
+) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+--
+-- table structure for table `subjects`
+--
+
+/*!40101 set @saved_cs_client     = @@character_set_client */;
+/*!40101 set character_set_client = utf8 */;
+CREATE TABLE `subjects` (
+  `subject` varchar(255) COLLATE latin1_general_cs NOT NULL,
+  `version` int(11) NOT NULL,
+  `schema_id` int(11) NOT NULL,
+  `project_id` int(11) NOT NULL,
+  `created_on` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  PRIMARY KEY (`subject`, `version`, `project_id`),
+  KEY `project_id_idx` (`project_id`),
+  KEY `created_on_idx` (`created_on`),
+  CONSTRAINT `project_idx` FOREIGN KEY (`project_id`) REFERENCES `project` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION,
+  CONSTRAINT `schema_id_idx` FOREIGN KEY (`schema_id`, `project_id`) REFERENCES `schemas` (`id`, `project_id`) ON DELETE NO ACTION ON UPDATE NO ACTION
+) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `subjects_compatibility`
+--
+CREATE TABLE `subjects_compatibility` (
```

Comment:
it is not a field in `subjects` because there can be multiple versions of a subject but a common compatibility for all of them.
it does not have a constraint to `subjects` because one row in `subjects_compatibility` can refer to multiple rows in `subjects`. A foreign key identifies uniquely a row in another table so it's not a good candidate.
`schemas` and `subjects_compatibility` don't have a constraint because they are not related to each other.